feat(content): New Missions: NPC escorts (#8206)
endless-sky/endless-sky: data/human/human missions.txt

				die
			label skip
			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`

> government "Escort (Betraying)"

Reviewer: ```suggestion
government "Escort (Betraying)"
```

I'm not sure, but I think we may able to use the same merchant spoof government here as in the disguised warlord missions?

Author: Setting that government to hostile in this mission would break those missions, since it'd mean that in subsequent missions the disguised pirates would start as hostile.

					decline
			`	"Alright, let's go! Don't forget to pick up a full cargo of luxury goods before setting off. Oh, and if you didn't already know, you can get to <planet> from here in just four jumps. North, west, southwest, then south."
				accept
			label otherplans
			`	"Alright, no worries, maybe another time?" The man smiles and walks away, into the spaceport.`
				defer
	on enter Moktar
		dialog `As expected, there are pirates waiting for you in Moktar. Unfortunately, the <npc> makes no attempt to attack them, instead turning its guns on you.`
		"reputation: Escort (Betraying)" <?= -1000
		fail
	on enter Menkar
		dialog `The <npc> hails you: "Wrong way, mate! It'll take you more than a week to get there if you go the whole way round. Head back up to Mirfak, then north, west, southwest and south."`
	on enter Zaurak
		dialog `The <npc> hails you again: "Figured it out, did you? Bit late for that now, mate. You're still going down."`
		"reputation: Escort (Betraying)" <?= -1000
		fail
	on abort
		dialog `The <npc> hails you: "Figured it out, did you? Bit late for that now, mate. You're still going down."`
		"reputation: Escort (Betraying)" <?= -1000
	npc

> 		government "Pirate"

Reviewer: ```suggestion
		government "Pirate"
```
If these are generic pirates, other ships will attack them even though they're not attacking the deceiver after the switch.

It may be best to have them also be the betraying government?

	description "The captain of the <npc> has offered to fly alongside you, protecting you from pirates, on your way to <destination>."
	source
		or
			government "Free Worlds"
			attributes "dirt belt"
	destination
		government "Free Worlds" "Syndicate" "Republic"
		distance 5 8
	to offer
		"armament deterrence" <= 2
		"cargo attractiveness" >= 2
		random < 15
	on offer
		conversation
			`As you are walking from your landing pad to the spaceport, you pass a cheerful man standing next to an impressively well-armed Bastion. He notices you, waves to you, and says "Hey, need some help with the local pirates? I'm on my way to <destination> at the moment, and I'd be happy to fly alongside your ship on the way for 15,000 credits."`
			choice
				`	"Thank you, that would be perfect!"`
				`	"Thanks for the offer, but I'm not headed that way."`
					goto thanksbut
			`	The man smiles as you hand over the credits. "Sure thing! We'll head out tomorrow."`
				accept
			label thanksbut
			`	The man nods. "Fair enough, you just keep yourself safe out there, alright?"`
			`	As you walk away, you see him make the same offer to the passing captain of a Hauler.`
				decline
	on accept

> 		payment -15000

Reviewer 2: I think you should add a daily cost, too (within an on daily)

Author: I may be misunderstanding what the personalities do, but I thought timid would make him stick close to you rather than running to the other side of the system and leaving you behind.

Reviewer 2: yeah true, keep it
~~should it be coward?~~

				accept
	on enter Moktar
		dialog `As expected, there are pirates waiting for you in Moktar. Unfortunately, the <npc> makes no attempt to attack them, instead turning its guns on you.`
		"reputation: Escort (Betraying)" <?= -1000
		fail
	on enter Menkar
		dialog `The <npc> hails you: "Wrong way, mate! It'll take you more than a week to get there if you go the whole way round. Head back up to Mirfak, then north, west, southwest and south."`
	on enter Zaurak
		dialog `The <npc> hails you again: "Figured it out, did you? Bit late for that now, mate. You're still going down."`
		"reputation: Escort (Betraying)" <?= -1000
		fail
	on enter Oblate Cardax Volax Nihal Elnath Acamar Kugel Algol Hamal
		dialog `You receive a message from the <npc>, which is now several systems away: "Hello? You still there? I don't know what just happened, but on the off chance you're still alive, I thought I'd just let you know that the deal's off, don't worry about the payment."`
		fail

> 	on abort
> 		dialog `The <npc> hails you: "Figured it out, did you? Bit late for that now, mate. You're still going down."`
> 		"reputation: Escort (Betraying)" <?= -1000

Reviewer 3: This dialog doesn't make sense if you trigger it on-world.
```suggestion
```

Reviewer 2: we can just have it be a conversation that checks for "flagship landed", then

				die
			label skip
			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`

> government "Escort (Betraying)"

Author: Placing a government that's only used by a single mission in the main governments file seems unhelpful - it makes the governments file larger and thus less convenient to look through, and it also means you need to look in two separate files to find out all the necessary information about the mission, rather than only one.

> government "Escort (Betraying)"

Reviewer 2: I agree with you but thats the current convention, and changing that should be in another PR

	"display name" "Merchant"
	swizzle 5
	
	"player reputation" 10
	"attitude toward"
		"Korath" -.01
	"fine" 0
	"friendly hail" "friendly civilian"
	"hostile hail" "hostile pirate"

mission "Escort offer (Betrayal)"
	landing
	name "Escort to <planet>"
	description "The captain of the <npc> has offered to fly alongside you, protecting you from pirates, on your way to <destination>. In exchange, he has asked for 80,000 credits upon your arrival."

> 	source Sunracer

Reviewer 3: ```suggestion
	deadline
	"apparent payment" 280000
	cargo "sealed barrels" 220
	source Sunracer
```

	destination Maelstrom

> 	to offer

Reviewer 3: To stop players with a jump drive from breaking this mission:
```suggestion
	to offer
		not "outfit: Jump Drive"
```

Author: I thought of this, hence the dialogue that occurs if you reach systems you can only get to by JD from where the mission offers. It might make sense though if once the main campaign is completed, people become aware of JDs and would hence notice them, so maybe if the main plot is complete and you have a JD it shouldn't offer.

		"armament deterrence" <= 2

> 		"cargo attractiveness" >= 2.8

Author: Changed to 6, because the 2.8 was calculated using the wrong formula.

> 	on offer

Reviewer 3: Just in case we use "Escort (Betraying)" elsewhere.
```suggestion
	on offer
		"reputation: Escort (Betraying)" = 10
```

		conversation

> 			`Almost immediately after you land, a heavily modified Firebird with oversized guns follows you down, landing on the pad next to you. A young man with short hair steps out, walks over to your ship, and says, "Hey, you've got a lot of cargo space there. I bet you make loads of dough with it, trading or running jobs. Say, luxury goods are quite cheap here, aren't they? I reckon you'd make an absolute shedload running 'em from here to <planet>. At least 200k, I'll bet. It's in <system>, if you're not familiar with it. Lot of pirates round there, though. I can fight 'em, but there's not much money to be made hauling cargo in this thing." He gestures to his ship. "Tell you what, for 40%, I'll see to it that you arrive there safely. What do you think?"`

Reviewer 3: ```suggestion
			`As you land, a heavily modified Firebird with oversized guns follows you down, settling on the pad next to you. A young man with short hair steps out, walks over to your ship, and says, "Hey, you've got a lot of cargo space there. I bet you make loads of dough with it, trading or running jobs. Say, I know somebody who could use a ship like yours. He needs to get <tons> of cargo to <destination> by <day>, you see. Payment's about 350,000 credits. Lot of pirates round there, though. I can fight 'em, but there's not much money to be made hauling cargo in this thing." He gestures to his ship. "Tell you what, for 20% of the payment, I'll see to it that you arrive there safely. What do you think?"`
```

			choice

> 				`	"I was just thinking of doing the same, glad to know I'll have you with me!"`

Reviewer 3: ```suggestion
```

Reviewer 4: Alternatively, add a new response following the suggested changes in line 5458.

> 				`	"Sounds like a plan. 80,000 credits, then?"`

Reviewer 3: ```suggestion
				`	"Sounds like a plan. You'll be taking 70,000 credits, then?"`
```

				`	"Actually, I have other plans. Sorry."`
					goto otherplans
				`	"Sorry, that just sounds too dangerous for me."`
					decline

> 			`	"Alright, let's go! Don't forget to pick up a full cargo of luxury goods before setting off. Oh, and if you didn't already know, you can get to <planet> from here in just four jumps. North, west, southwest, then south."`

Reviewer 3: ```suggestion
			`	"That's right. Now, let's go! The cargo'll be loaded on your ship by sundown. Oh, and if you didn't already know, you can get to <planet> from here in just four jumps. North, west, southwest, then south."`
```

				accept
			label otherplans
			`	"Alright, no worries, maybe another time?" The man smiles and walks away, into the spaceport.`
				defer
	on enter Moktar
		dialog `As expected, there are pirates waiting for you in Moktar. Unfortunately, the <npc> makes no attempt to attack them, instead turning its guns on you.`
		"reputation: Escort (Betraying)" <?= -1000
		fail
	on enter Menkar

> 		dialog `The <npc> hails you: "Wrong way, mate! It'll take you more than a week to get there if you go the whole way round. Head back up to Mirfak, then north, west, southwest and south."`

Reviewer 3: ```suggestion
		dialog `The <npc> hails you: "Wrong way, mate! You'll miss the deadline if you go the whole way round. Head back up to Mirfak, then north, west, southwest and south."`
```

	on enter Zaurak

> 		dialog `The <npc> hails you again: "Figured it out, did you? Bit late for that now, mate. You're still going down."`

Reviewer 3: The whole point of the mission was to lure you into a vulnerable position, away from law enforcement. Why would he then attack you in a system protected by the Syndicate?

Author: Zaurak is only one jump away from Alcyone, and has fairly similar frequencies of pirate and Syndicate fleets, so it's the best opportunity to attack if you're going round the other way.
```
	fleet "Small Syndicate" 1600
	fleet "Large Syndicate" 2500
	fleet "Small Core Pirates" 2000
	fleet "Large Core Pirates" 4000
```

Reviewer 3: He could also just give up on attacking you?

		"reputation: Escort (Betraying)" <?= -1000
		fail
	on enter Oblate Cardax Volax Nihal Elnath Acamar Kugel Algol Hamal

> 		dialog `You receive a message from the <npc>, which is now several systems away: "Hello? You still there? I don't know what just happened, but on the off chance you're still alive, I thought I'd just let you know that the deal's off, don't worry about the payment."`

Reviewer 3: ```suggestion
		dialog `You receive a message from the <npc>, which is now several systems away: "Hello? You still there? I don't know what just happened, but on the off chance you're still alive, I thought I'd just let you know that the deal's off." You open up the cargo that he wanted you to carry, and find nothing but worthless barrels of water.`
```

		fail
	on abort
		dialog `The <npc> hails you: "Figured it out, did you? Bit late for that now, mate. You're still going down."`
		"reputation: Escort (Betraying)" <?= -1000
	npc

> 		government "Escort (Betraying)"

Reviewer 3: Shouldn't this just be a regular pirate?

Author: I had them as a regular pirate originally, it was suggested that they should use the same government.

Reviewer 3: That doesn't make sense, though? It's even mentioned in the mission that you see a pirate vessel, not a pirate vessel disguised as a merchant.
